src/screens/Wallet/Send/Modals/ConfirmSendModal/ConfirmSendModalContainer.js

```diff
@@ -10,16 +10,19 @@ import { bindActionCreators } from 'redux'
 import { selectCurrentCurrency } from '@chronobank/market/redux/selectors'
 import { getBitcoinCurrentWallet } from '@chronobank/bitcoin/redux/selectors'
 import { requestBitcoinSendRawTransaction } from '@chronobank/bitcoin/service/api'
+import { getCurrentWallet } from '@chronobank/session/redux/selectors'
 import ConfirmSendModal from './ConfirmSendModal'
 
 const mapStateToProps = (state) => {
+  const masterWalletAddress = getCurrentWallet(state)
+
   return {
     prices: {
       BTC: {
         USD: 1499,
       },
     },
```

Comment:
I could delete this and use prices selector from market

Reply:
It will be great.